packages/wdio-cucumber-framework/src/index.ts

```diff
@@ -43,7 +43,11 @@ const { incrementing } = IdGenerator
 
 function getResultObject (world: ITestCaseHookParameter): Frameworks.PickleResult {
     return {
-        passed: world.result?.status === Cucumber.Status.PASSED,
+        passed: (
+            world.result?.status === Cucumber.Status.PASSED
+            || world.result?.status === Cucumber.Status.SKIPPED
+            || world.result?.status === Cucumber.Status.PENDING
```

Comment:
```
Pending

When a step definition’s method or function invokes the pending method, the step is marked as pending (yellow, as with undefined ones), indicating that you have work to do. If you use --strict, this will cause Cucumber to exit with 1.
```
It's a bit unclear if this should really be passing, I don't think it will do much harm but it's good to think about it here :)

Not a blocker for me

Reply:
I just noticed the following:
```
Undefined

When Cucumber can’t find a matching step definition, the step gets marked as undefined (yellow), and all subsequent steps in the scenario are skipped. If you use --strict, this will cause Cucumber to exit with 1.
Pending

When a step definition’s method or function invokes the pending method, the step is marked as pending (yellow, as with undefined ones), indicating that you have work to do. If you use --strict, this will cause Cucumber to exit with 1.
```
We handle `undefined` as a failure but we do not for `pending` while it seems like they should be treated the same, this makes me believe that `passed` should not be set to true.
What do you think?

Reply:
yeah, it's a bit of a grey area, I'll revert that one so we don't introduce something breaking. Good call @erwinheitzman